draft-ietf-quic-recovery.md

```diff
@@ -1142,6 +1142,22 @@ Pseudocode for OnPacketSent follows:
      SetLossDetectionTimer()
 ~~~
 
+## On Receiving a Datagram
+
+Receiving a datagram might allow the server to send if the
+server was previously blocked by the anti-amplification limits.
+This is necessary even if none of the packets in the datagram
+are successfully processed.
```

Comment:
```suggestion
When a server is  blocked by anti-amplification limits, receiving
a datagram unblocks it, even if none of the packets in the datagram
are successfully processed. In such a case, the PTO timer is now set,
since data is in flight and probes can be sent.
```